project-reports/2024/2024-annual-Hyperledger-Indy.md

```diff
@@ -0,0 +1,98 @@
+
+The Hyperledger Indy ecosystem community came together at a series of Hyperledger Indy Ecosystem Summits (organized by the Sovrin Foundation), bringing together network operators, node operators, project maintainers, network users, vendors to discuss the state of Indy and proposals for moving the project forward. The summits were well-attended (up to 90 participants), a better understanding of the community of interest was gained, and next steps were proposed, with some (mostly notably, the Indy-Besu initiative) acted upon.
+
+## Maintainer Diversity
+
+_Guidance: How many maintainers do you have, and which organisations are they from? How has the maintainers and diversity of your maintainers changed in the past year? Has the number of active maintainers increased/decreased? Has the diversity of maintainers increased/decreased? Please include a link to your existing [MAINTAINERS file](https://github.com/hyperledger/toc/blob/gh-pages/guidelines/MAINTAINERS-guidelines.md) and the MAINTAINERS file from last year (if appropriate). This is a good opportunity to ensure that your MAINTAINERS file is up to date and to retire any maintainers._
+
+There are 35 individuals on 22 Indy GitHub Teams representing at least 16 organizations. We continue to have participation from (more or less) the same organizations as in 2022 — which is both good (still participating) and bad (few newcomers).
+
+From the last Quarterly Report: _Still needed -- a cleanup of the teams are needed as some of the individuals are no longer active in the community. No progress was made on that cleanup this quarter. We have the team lists, but have not queried those on the list to see if they are still interested in being maintainers._
+
+Typical Indy [Maintainers File](https://github.com/hyperledger/indy-plenum/blob/main/MAINTAINERS.md).
+
+## Project Adoption
+
+_Guidance: What do you know about adoption, and how has this changed since your last review or since being accepted into Hyperledger Foundation? If you can list companies that are adopters of your project, please do so. Feel free to link to an existing ADOPTERS file if appropriate._
+
+Insights from the Hyperledger Indy Ecosystem Summit showed that project adoption is “stable”. It seems those that have adopted Indy continue to stay with it and while there is evidence of new private deployments, we’re not aware of new public ones. There remains little competition for what Indy is today — an easy to deploy, stable, token-less, public, permissioned blockchain that is purpose-built for privacy-preserving verifiable credentials. The best alternative to Indy, and one being investigated as a potential Indy replacement, is Besu, with features that align with the existing Indy concepts.
```

Comment:
See above comment - these can definitely be linked in a way to boost both projects.

Reply:
Just to add to what this report says and what Matt has noted: there's an ongoing feature being developed (in a mentorship project) within core Cacti that necessitates a dependency to Indy.

Two questions related to this (putting on my hat as a mentor as well as a reviewer):
- As you have highlighted above, there is relatively little competition for Indy in its niche, and that is what a lot of adopters and users find compelling about it. But oftentimes those users may need additional pluggable features (as core Indy is quite generic). Are there established guidelines and procedures for people wishing to add such features to Indy?
- Since the big refactoring of Indy and Aries, there seems to be relatively little guideline for people who were familiar with the older repositories (including `indy-sdk`) to migrate to the new ones. Is Discord the only way to get such guidance or are there more "static" references people can use?